app/javascript/utils/form/validations.js

```diff
@@ -102,4 +102,26 @@ validate.any = (...fns) => (value) => {
       map(fns, fn => fn(value))))
 }
 
+validate.isValidDOB = (person, errors, isPrimaryApplicant = false) => {
```

Comment:
@software-project how would you feel about using an options object for the `isPrimaryApplicant` argument? e.g. `opts = {}`. then the call is `isValidDOB(person, errors, {isPrimaryApplicant: true})`, which indicates what that last argument is for, instead of `isValidDOB(person, errors, true)`, which doesn't indicate what `true` will do and makes the dev have to go look up the function